src/actions/Reports/index.js

```diff
@@ -27,4 +27,21 @@ export const addCurrencyReport = (curr) => {
                 dispatch(toggleFetchingStatus());
             });
     }
+}
+
+export const selectCurrencyReport = (report) => {
+    return dispatch => {
+        dispatch({
+            type: types.SELECT_REPORT,
+            payload: report
+        });
+    }
+}
+
+export const deleteSelectedReport = () => {
+    return dispatch => {
+        dispatch({
+            type: types.DELETE_REPORT
+        })
+    }

```

Comment:
Good use of dispatch within Redux actions. Basically passing object as an argument to dispatch function is only shorthand, but makes code more readable & consistent.